public/js/pimcore/element/abstract.js

```diff
@@ -308,7 +312,7 @@ pimcore.element.abstract = Class.create({
 
         return [
             {
-                text: t("metainfo_copy_id"),
+                text: t("id") + " " + metainfo.id + ' - ' + t("copy"),
```

Comment:
so the translations will not work anymore for this key. right?

Reply:
Not sure what you mean, here. The key "metainfo_copy_id" is no longer in use. So it should be removed from the translations files, right?

Reply:
Ok. then we should remove translation key from `en.yaml`